bugbug/repository.py

```diff
@@ -1543,6 +1550,69 @@ def trigger_pull() -> None:
     trigger_pull()
 
 
+def get_diff(repo_path, original_hash, fix_hash) -> bytes:
+    client = hglib.open(repo_path)
+
+    current_rev = client.identify(id=True)
+
+    try:
+        client.rawcommand([b"shelve"])
+    except Exception:
+        pass
```

Comment:
We should try to be more specific with the exception type. This could hide problems and make it harder to debug. I suggest specifying the exact error that we are OK with without crashing the script. Also, it may be helpful to log such cases.

Reply:
Done in ae28dcffba2c5594a1f8a184c6688c2d8d35f41b!